sdk/client.go

```diff
@@ -58,7 +58,7 @@ func (cli *Client) GetRootTopicsPublic(ctx context.Context, reqHeaders Headers)
 	path := fmt.Sprintf("%s/topics", cli.hcCli.URL)
 
 	b, apiErr := cli.callTopicAPI(ctx, path, http.MethodGet, reqHeaders, nil)
-	if apiErr != nil {
+	if apiErr.Error() != "nil" {
```

Comment:
(question): why do you have to check the custom err nil condition like this ? just apierr != nil should do ?